src/oc_erchef/raml-docs/schemas/common.json

```diff
@@ -36,9 +36,31 @@
     "description": "A version constraint. A series of comma separated constraints, each of which may include VERSION, \"> VERSION\", \"< VERSION\", \">= VERSION\", \"<= VERSION\", \"= VERSION\" and \"~> VERSION\".",
     "type": "string"
   },
+  AclUpdatePermission: {
```

Comment:
Will fix, thanks .  The raml parser we're using doesn't actually resolve references to common.json, which is likely how it got through.